src/TLuaInterpreter.cpp

```diff
@@ -187,7 +188,7 @@ TLuaInterpreter::TLuaInterpreter(Host* pH, const QString& hostName, int id) : mp
 
     initLuaGlobals();
 
-    purgeTimer.start(2000);
+    purgeTimer.start(2min);
```

Comment:
Um, `2000` milliseconds is not 2 minutes!
```suggestion
    purgeTimer.start(2s);
```

Reply:
You're right but what does this even do?